code/src/index.css

```diff
@@ -1,13 +1,362 @@
+	margin-top: 20px;
+	font-family: "Grenze Gotisch", serif;
+}
+
+.previous-steps-header {
+	font-size: 16px;
+	text-align: center;
+	font-weight: normal;
+	margin-bottom: 0;
+}
+
+.previous-steps-list {
+	list-style: none;
+	display: flex;
+	flex-direction: row-reverse;
+	padding: none;
+	padding: 0;
+	justify-content: center;
+	margin-top: 0;
+}
+
+.previous-steps-list > li {
+	border: 1px solid #727272;
+	padding: 4px;
+	margin: 3px;
+	width: 31px;
+	border-radius: 50%;
+	height: 31px;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	font-size: 14px;
+	font-family: "Unica One", serif;
+}
+
+/*ACTION*/
+.action-wrapper {
+	width: 100%;
+	border: 1px solid #4f585d;
+	padding: 10px;
+	margin: 5px 0px;
+	background-color: rgb(79 88 93 / 38%);
+}
+
+.gameboard-action-button,
+.gameboard-end-button {
+	width: 100%;
+	height: 30px;
+	cursor: pointer;
+	background-color: #354f52;
+	color: #fff;
+	border: none;
+	font-size: 21px;
+	font-family: "Grenze Gotisch", serif;
+}
+
+.startboard-start-button {
+	width: 100%;
+	height: 45px;
+	max-width: 350px;
+	cursor: pointer;
+	border: none;
+	font-size: 21px;
+	font-family: "Grenze Gotisch", serif;
+	background-color: #475736;
+	color: #fff;
+}
+
+.gameboard-end-button {
+	background-color: #262f34;
+}
+button:hover {
+	background-color: #1d2a37;
+	color: #fff;
+}
+
+.gameboard-game-over-description {
+	font-size: 32px;
+	text-align: center;
+	color: #e6a348;
+	max-width: 540px;
+	align-self: center;
+}
+
+.gameboard-game-over-sub-description {
+	text-align: center;
+	font-size: 28px;
+	margin: 10px;
+}
+.action-description {
+	font-size: 19px;
+	font-weight: normal;
+	font-family: "Grenze", serif;
+	margin-top: 0;
+}
+
+/*MAP*/
+.row {
+	display: flex;
+}
+
+.cell,
+.cell-current-position {
+	background-color: #262d31;
+	width: 30px;
+	height: 30px;
+	border: 1px solid #727272;
+	border-radius: 50%;
+	align-items: center;
+	justify-content: center;
+	display: flex;
+}
+
+.cell-id {
+	font-size: 16px;
+	color: #fff;
+	font-family: "Unica One", serif;
+}
+
+.cell-current-position {
+	background-color: #e6a348;
+}
+
+.cell-current-position > .cell-id {
+	color: #262f34;
+	font-weight: bold;
+}
+
+.maze-map {
+	display: flex;
+	flex-direction: column;
+	margin-left: 38px;
+}
+
+.map-wrapper {
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	margin-top: 15px;
+	flex-direction: row;
+}
+
+.compass-wrapper {
+	display: flex;
+	align-items: center;
+}
+
+.game-compass {
+	font-size: 13px;
+	margin-left: 10px;
+}
+
+/*LOADER*/
+.loader {
+	width: 80px;
+	height: 80px;
+	margin: auto;
+}
+.loader:after {
+	content: " ";
+	display: block;
+	width: 64px;
+	height: 64px;
+	margin: 8px;
+	border-radius: 50%;
+	border: 6px solid #fff;
+	border-color: #fff transparent #fff transparent;
+	animation: loader 1.2s linear infinite;
+}
+@keyframes loader {
+	0% {
+		transform: rotate(0deg);
+	}
+	100% {
+		transform: rotate(360deg);
+	}
+}
+
+@media only screen and (min-width: 600px) {
```

Comment:
Nice styling on this media-query! We like for example the action-wrapper styling, looked nice in the browser 👍